contracts/gateway-v1.yaml

```diff
@@ -3579,4 +3694,66 @@ components:
         resp:
           $ref: '#/components/schemas/RequestReference'
       xml:
-        name: PatientIdentificationResponse
\ No newline at end of file
+        name: PatientIdentificationResponse
+    HipInitiatedLinkRequest:
+      type: object
+      properties:
+        requestId:
+          type: string
+          format: uuid
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        cmId:
+          type: string
+        patient:
+          type: object
+          properties:
+            id:
+              type: string
+            careContexts:
+              type: array
+              items:
+                $ref: '#/components/schemas/HipInitiatedLinkCareContext'
+              xml:
+                name: careContexts
+                wrapped: true
+          xml:
+            name: HipInitiatedLinkRequest
+    HipInitiatedLinkResult:
```

Comment:
we need a payload/result as root atttibute. 
maybe something like :
"linkRequest": {
    "sessionStatus": "SUCCESS|FAILURE"
 }

like we have in /v1/health-information/hip/on-request

Reply:
you need to define sessionStatus as enum (possible values SUCCESS and FAILURE)